packages/compose/src/hooks/use-resize-observer/index.tsx

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import type { RefCallback, RefObject } from 'react';
+import type { RefCallback, RefObject, ReactElement } from 'react';
```

Comment:
Nit: if we're sorting the inline imports alphabetically, `ReactElement` would go first:

```suggestion
import type { ReactElement, RefCallback, RefObject } from 'react';
```